celestia/peers.txt

```diff
@@ -1 +1,3 @@
-
+6ba9f87efe6beb23bb7af2204eac769767d8ec78@78.129.165.127:23656
+491c37d5de210600463c5599a3ffd272c8690014@57.128.20.184:23656
+7f067f54987cccfe489fbe302ff1bf6c56aaf02f@162.19.81.54:23656
```

Comment:
Could you instead please follow the following pattern (then we don't need to resolve merge conflicts):

Please make the peers and gentx separate files, so that your PR looks like this:

`celestia/gentx/validatorname.json // 
celestia/validatorname.txt //`

Afterwards we will consolidate all the peers from your txt files to peers.txt. Reason we ask is because merging PRs the way they are now is slow and problematic. Thank you for your understanding!

Reply:
done